specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/examples/FleetMembers_Create.json

```diff
@@ -0,0 +1,66 @@
+{
+  "parameters": {
+    "api-version": "2022-09-02-preview",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "fleetName": "fleet-1",
+    "fleetMemberName": "member-1",
```

Comment:
>fleetMemberName [](http://example.com/codeflow?start=5&length=15)

want to include sample etag headers for fun? :)

Reply:
Thanks Tim for the review. The only change in this version from the last one (0702 api) should be to add a regex. It's a 3 lines change, so I would keep the diff as small as possible.